src/wedderburn_decomposition.jl

```diff
@@ -173,15 +173,15 @@ end
 function invariant_vectors(
     tbl::Characters.CharacterTable,
     act::Union{<:ByPermutations,<:BySignedPermutations},
-    basis::StarAlgebras.Basis,
+    basis::StarAlgebras.ExplicitBasis,
 )
     return invariant_vectors(parent(tbl), act, basis)
 end
 
 function invariant_vectors(
     G::Group,
     act::ByPermutations,
-    basis::StarAlgebras.Basis{T,I},
+    basis::StarAlgebras.ExplicitBasis{T,I},
 ) where {T,I}
     tovisit = trues(length(basis))
     invariant_vs = Vector{SparseVector{Rational{Int},I}}()
```

Comment:
The line below fails because `eachindex(basis)` is not defined for `FixedBasis`

Reply:
It was broken by https://github.com/JuliaAlgebra/StarAlgebras.jl/pull/21/commits/7f17590e35ec15d0748bb2dd4972f122e06f43e8 I believe